src/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Footer(props){
+  const enigmaBay = <a href='https://github.com/EnigmaBay' className='themed-hyperlink' data-theme={props.dataTheme}>EnigmaBay</a>;
+  const mit = <a href='https://github.com/EnigmaBay/lingobingojs/blob/main/LICENSE' className='themed-hyperlink' data-theme={props.dataTheme}>MIT License</a>;
+  return (
+    <div data-theme={props.dataTheme} className='text-center themed-footer'>
```

Comment:
Try adding a 'rounded-4' here, or updating themed-footer (in dev-theme.css) to get the rounded edges like in the other text areas on the page.